app/ios/iosinterface.mm

```diff
@@ -167,4 +256,34 @@ -( void )showImagePicker:( int )sourceType : ( IOSImagePicker * )handler
 
   }
 }
+
+- ( NSString * ) readExif:( NSString * ) imageFileURL : ( NSString * )tag
+{
+  NSObject *result = readExifAttribute( imageFileURL, tag );
+  if ( !result )
+  {
+    return nil;
+  }
+
+  if ( [result class] == [NSString class] )
+  {
+    return ( NSString * ) result;
+  }
+  else if ( [result class] == [NSNumber class] )
+  {
+    NSString *stringResult = [( NSNumber * )result stringValue];
+    return stringResult;
+  }
+  else if ( [result class] == [NSDecimalNumber class] )
+  {
+    NSString *stringResult = [( NSDecimalNumber * )result stringValue];
+    return stringResult;
+  }
+  else
+  {
+    NSString *stringResult = [NSString stringWithFormat:@"%@", result];
+    return stringResult;
+  }
+}
+
```

Comment:
if various tags returns always same NS type, I would prefer to have 1 function for each tag type? since now you convert native type -> NSString and than in C++ QString -> nativetype...

Reply:
This conversion is not ideal indeed. We can have a map of `tag` -> `type` and handle result according it, but the map would be rather long and maybe not complete at some point soon.

What can be quite a good solution is that for both Android and iOS (or other platforms later on), we can omit conversion to string and directly create QVariant out of native type.